GameServerCore/Domain/GameObjects/Spell/ISpell.cs

```diff
@@ -52,7 +52,7 @@ public interface ISpell: IUpdate
         /// <summary>
         /// Whether or not the script for this spell is the default empty script.
         /// </summary>
-        bool HasEmptyScript { get; }
+        bool HasEmptyScript { get; set; }
```

Comment:
You shouldn't expose this property to be public.

Reply:
Resolved